tests/baselines/reference/classExtendingPrimitive.errors.txt

```diff
@@ -14,13 +14,13 @@ tests/cases/conformance/classes/classDeclarations/classHeritageSpecification/cla
     
     class C extends number { }
                     ~~~~~~
-!!! error TS2693: 'number' only refers to a type, but is being used as a value here.
+!!! error TS18042: An interface cannot extend a primitive type like 'number'; An interface can only extend named types and classes.
     class C2 extends string { }
                      ~~~~~~
-!!! error TS2693: 'string' only refers to a type, but is being used as a value here.
+!!! error TS18042: An interface cannot extend a primitive type like 'string'; An interface can only extend named types and classes.
     class C3 extends boolean { }
                      ~~~~~~~
-!!! error TS2693: 'boolean' only refers to a type, but is being used as a value here.
+!!! error TS18042: An interface cannot extend a primitive type like 'boolean'; An interface can only extend named types and classes.

```

Comment:
You're reporting this on a class, not an interface.

Reply:
Changes:
* Two errors to distinguish between class and interface 
* Conditional flow using bitwise operations on `meaning & class` and `meaning & interface`